src/modules/restaurants/api.ts

```diff
@@ -1,21 +1,19 @@
 import axios from 'axios';
 
+const BASE_URL = process.env.REACT_APP_BASE_URL;
+
+
 export async function getRestaurantsFromAPI(filters: any) {
 	const result = await axios.get(
-		`http://localhost:5000/restaurants?minPrice=${filters.priceRange.min}&maxPrice=${filters.priceRange.max}&distance=${filters.distance}&category=${filters.category}&rating=${filters.rating}`
+		`${BASE_URL}/restaurants?minPrice=${filters.priceRange.min}&maxPrice=${filters.priceRange.max}&distance=${filters.distance}&category=${filters.category}&rating=${filters.rating}`
 	);
 
 	return result;
 }
 
-export async function getRestaurantByIDFromAPI(id: any) {
-	const result = await axios.get(`http://localhost:5000/restaurants/${id}`);
+export async function getRestaurantByNameFromAPI(name: any) {
+	const result = await axios.get(`${BASE_URL}/restaurants/${name}`);
```

Comment:
even though i said oits ok to use restaurants names, id is the better way since two restaurants could have the same name